### PR TITLE
replaced referenced to firehol github org with netdata github org

### PR DESCRIPTION
**Pull request by ktsaou**
TO CHECK:

- [ ] third party badges we use on README.md
- [ ] sourceforge integration
- [ ] packaging/README.md 
- [ ] activate coverity for netdata/netdata

### CODE REVIEW

File: coverity-scan.sh [hunk 1]
```diff
@@ -44,4 +44,4 @@ curl --progress-bar --form token="${token}" \
   --form file=@netdata-coverity-analysis.tgz \
   --form version="${version}" \
   --form description="netdata, real-time performance monitoring, done right." \
-  https://scan.coverity.com/builds?project=firehol%2Fnetdata
+  https://scan.coverity.com/builds?project=netdata%2Fnetdata
```
Comment (paulfantom) [inline coverity-scan.sh]: Not active yet

Comment (ktsaou) [reply coverity-scan.sh]: added it to the list above

Comment (paulfantom) [reply coverity-scan.sh]: It is already on the list here: https://github.com/netdata/netdata/issues/4190#issuecomment-422779381 :)

Comment (paulfantom) [reply coverity-scan.sh]: Let's not duplicate lists

**Comment by paulfantom** [comment]
relates to #4249

**Pull request merged**